src/views/dataset/DatasetInfo.vue

```diff
@@ -285,21 +285,21 @@ export default class DatasetInfo extends Vue {
       return;
     }
 
-    const date = new Date();
     try {
+      const dataset = this.store.dataset;
+      if (dataset === null) {
+        throw new Error("dataset was null when it shouldn't be");
```

Comment:
I guess you should put a simpler error message: "Dataset not set"

Reply:
That some code I only moved around
I'll change both error message for "Dataset not set" and "Config not set"